Arithmetic_kernel/test/Arithmetic_kernel/CMakeLists.txt

```diff
@@ -27,16 +27,13 @@ if(GMP_FOUND)
     # version needs GMP>=4.2, so we require this dependency only here and
     # not in FindMPFI.cmake
     if(_IS_GMP_VERSION_TO_LOW)
-      message(
-        STATUS
-          "MPFI tests need GMP>=4.2, some of the tests will not be compiled")
+      message(STATUS "NOTICE: MPFI tests need GMP>=4.2, some of the tests will not be compiled")
```

Comment:
According to the description of the PR:
>Make it so message() calls are more consistent: if it's non-essential, it's a STATUS; if it's important, it's a NOTICE.

But what is this `STATUS` type with text `NOTICE`?

Reply:
It indicates that a third party dependency is missing, but this dependency is only required for *some* of the targets. The `NOTICE:` is there to make it more visible since CMake can sometimes dump a lot of info.